index.html

```diff
@@ -0,0 +1,56 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Best-ToDo-App</title>
+    <link rel="stylesheet" href="style.css">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:ital,wght@0,100;0,400;1,400&display=swap" rel="stylesheet">
+    <script src="https://kit.fontawesome.com/75ec349a37.js" crossorigin="anonymous"></script>
+</head>
+<body class="body">
+    <header class="header">
+        <div class="pavadinimas">
+        <h1>Best <span>ToDo</span> App</h1>
+        </div>
+        <div class="icons">
+            <a href="https://www.facebook.com/"><i class="fa-brands fa-facebook"></i></a>
+            <a href="https://twitter.com/"><i class="fa-brands fa-twitter"></i></a>
+            <a href="https://www.linkedin.com/"><i class="fa-brands fa-linkedin"></i></a>
+        </div>
+    </header>
+    <main>
+        <div class="task-input">
+            <h2 class="your-tasks">Your tasks</h2>
+            <form action="" class="form"> 
+            <label for="task"></label>
+            <input type="text" id="task" placeholder="e.g. wash dishes">
+            <input class="add" type="submit" value="Add">
+            </form>
+            <p class="error">There Was An Error</p>
+        </div>
+        <div class="all-tasks">
+            <div class="task-output">
+                <h2 class="your-tasks-noted task-not-done">Take out Trashes</h2>
```

Comment:
Do we really need a separate task for a task not done? Perhaps the default styling should be for the all tasks not done and they would be updated by a class that's added only for tasks that are done